client/src/cmdhfmf.c

```diff
@@ -4364,9 +4376,9 @@ static int CmdHF14AMfCSetBlk(const char *Cmd) {
 static int CmdHF14AMfCLoad(const char *Cmd) {
 
     CLIParserContext *ctx;
-    CLIParserInit(&ctx, "hf mf cload",
-                  "Load magic gen1a card with data from (bin/eml/json) dump file\n"
-                  "or from emulator memory.",
+    CLIParserInit(&ctx, "hf mf cload\n",
+                  "Load magic gen1a card with data from (bin/eml/json) dump file or from emulator memory.\n"
+                  "Only works with magic gen1a cards",
```

Comment:
same cake on cake feeling here.   First state its for Gen1a,   then  repeat it next line. 
Here you also see on example where I tried breaking lines up.